bindings/python/test/interpolators/test_barycentric_rational.py

```diff
@@ -0,0 +1,33 @@
+################################################################################################################################################################
+
+# @project        Open Space Toolkit ▸ Mathematics
+# @file           bindings/python/test/interpolators/test_barycentric_rational.py
+# @author         Vishwa Shah <vishwa@loftorbital.com>
+# @license        Apache License 2.0
+
+################################################################################################################################################################
+
+from ostk.mathematics.curve_fitting.interpolation import BarycentricRational
+
+################################################################################################################################################################
+
+class TestBarycentricRational:
+
+    def test_default_constructor (self):
+        BarycentricRational([0.0, 1.0, 2.0, 4.0, 5.0, 6.0], [0.0, 3.0, 6.0, 9.0, 17.0, 5.0])
```

Comment:
I recommend testing the named args here too:

```suggestion
        BarycentricRational(x = [0.0, 1.0, 2.0, 4.0, 5.0, 6.0], y = [0.0, 3.0, 6.0, 9.0, 17.0, 5.0])
```